lib/hmpps_oauth_client.js

```diff
@@ -0,0 +1,22 @@
+const axios = require("axios");
+
+exports.getHMPPSAuthToken = async function () {
+  try {
+    const response = await axios.post(
+      "http://localhost:8090/auth/oauth/token?grant_type=client_credentials&scope=",
+      {},
+      {
+        headers: {
+          Authorization: `Basic ${Buffer.from(
+            "prisonstaffhubclient:clientsecret",
```

Comment:
Is `prisonstaffhubclient` intended at this point?

Reply:
I had a conversation with Andy Marke from the Community API team, and he sent me [this base64 encoded string](https://mojdt.slack.com/archives/CR5ESQ8T1/p1599723574056300?thread_ts=1599665934.052800&cid=CR5ESQ8T1) which contains `prisonstaffhubclient`. I felt it was better to put in the credentials themselves as it's (possibly?) a bit clearer.